src/python/turicreate/_cython/cy_sframe.pyx

```diff
@@ -375,3 +382,22 @@ cdef class UnitySFrameProxy:
     cpdef delete_on_close(self):
         with nogil:
           self.thisptr.delete_on_close()
+
+
+    def __reduce__(self):
+
```

Comment:
`self.materialize()`?

Reply:
The save action does that, again should be fine.